src/Microsoft.TestPlatform.Common/ExtensionFramework/Utilities/TestExtensions.cs

```diff
@@ -390,4 +485,42 @@ internal void InvalidateCache()
         }
     }
 
+    private void AddCachedExtensionToExtensionMap<T>(
```

Comment:
I would also rename this map.

Reply:
Done, made sure there's no more reference to the word "map" :)